docs/antora/modules/ROOT/pages/configuration/docker.adoc

```diff
@@ -9,338 +9,26 @@ Mona Bärenfänger <mona@lightcurve.io>
 :page-next-title: Docker commands
 
 :url_references_config: configuration/index.adoc
+:url_github_env: https://github.com/LiskHQ/lisk-service/blob/development/docker/example.env
+:url_github: https://github.com/LiskHQ/lisk-service
+:url_docker_commands: management/docker.adoc
 
-How to configure Lisk Service.
+On this page, we discuss how to customize the Docker installation of Lisk Service.
 
 == Configuring Lisk Service
+The {url_github}[Lisk Service] provides all possible Docker related configurations and their default values in an `example.env` file.
 
-Copy the example configuration options to `lisk-service/.env`:
+TIP: The default {url_github_env}[example.env] file can be seen on the GitHub repository.
 
-.lisk-service/
-[source,bash]
-----
-cp docker/example.env .env
-vim .env
-----
-
-TIP: Check the xref:{url_references_config}[] for a summary of all available environment variables to configure Lisk Service.
-
-Open the `.env` file to change the default configuration options of Lisk Service:
-
-.lisk-service/.env
-[source,bash]
-----
-## Common Docker configuration
-
-# Service broker
-# SERVICE_BROKER=redis://redis_common:6379/0
-
-# Lisk Service log config
-# SERVICE_LOG_CONSOLE=false
-# SERVICE_LOG_FILE=false
-# SERVICE_LOG_GELF=false
-# SERVICE_LOG_LEVEL=debug
-# SERVICE_LOG_STDOUT=true
-
-## Lisk Service Core
-
-# Lisk static assets, ie. known account lists
-LISK_STATIC=https://static-data.lisk.com
-
-# Lisk Service geolocation backend (empty = disabled)
-GEOIP_JSON=
-
-# Create search index (0 = full index)
-INDEX_N_BLOCKS=0
-
-# MySQL Snapshot settings
-
-# Enable or disable apply snapshot feature
-ENABLE_APPLY_SNAPSHOT=true
-# Custom snapshot url (Expected to end with sql.gz)
-INDEX_SNAPSHOT_URL='https://snapshots.lisk.io/mainnet/service.sql.gz'
-# When the MySQL is hosted as a docker-compose service, set the following environment variables:
-# Set docker-compose file path (use absolute path)
-DOCKER_COMPOSE_FILEPATH='/Users/lisk/lisk-service/jenkins/mysql/docker-compose.yml'
-# Set MySQL service name as defined in the above docker-compose file
-DOCKER_MYSQL_SERVICE_NAME=mysql
-
-# Use default genesis block height
-# GENESIS_HEIGHT=0
-# FEE_EST_DEFAULT_START_BLOCK_HEIGHT=0
-
-# Fee estimator
-ENABLE_FEE_ESTIMATOR_QUICK=true
-ENABLE_FEE_ESTIMATOR_FULL=false
-
-# Transaction statistics
-ENABLE_TRANSACTION_STATS=true
-TRANSACTION_STATS_HISTORY_LENGTH_DAYS=366
+To customize the Docker installation, copy the example configuration file to a `.env` file in the root of the `lisk-service` directory, where we have the `docker-compose.yml` file.
```

Comment:
imho, I would suggest we slightly reword this sentence as follows:
To customize the Docker installation, copy the example configuration file to a `.env` file in the root of the `lisk-service` directory, where the `docker-compose.yml` file is located.
```